assets/component-visual-display.css

```diff
@@ -82,6 +82,7 @@
   display: block;
   height: 100%;
   width: 100%;
+  forced-color-adjust: none;
```

Comment:
I was surprised this was working outside of a `@media` query for `forced-colors: active`, but I tophatted in Chrome and Edge and it all works. Thanks for the update.